src/view/editableelement.js

```diff
@@ -84,11 +84,10 @@ export default class EditableElement extends ContainerElement {
 			isFocused => isFocused && document.selection.editableElement == this
 		);
 
-		// Update focus state before each rendering. Rendering should not change neither the selection nor the value of
-		// document.isFocused property.
-		this.listenTo( document, 'render', () => {
+		// Update focus state based on selection changes.
+		this.listenTo( document.selection, 'change', () => {
```

Comment:
Can we use `view#change` event here?

Reply:
I would stay with `selection#change`. EditableElement's focus state depends on document focus and selection position, so it's updated whenever `document.isFocused` observable changes or view selection fires `#change` event.